themes/classic/_dev/js/listing.js

```diff
@@ -29,6 +29,8 @@ import 'velocity-animate';
 import ProductMinitature from './components/product-miniature';
 
 $(document).ready(() => {
+  let history = window.location.href;
```

Comment:
```suggestion
  const history = window.location.href;
```

Reply:
@NeOMakinG It's me or the change is not done ?

Reply:
@NeOMakinG The change is not ever done !